src/ui/styles/ContentToggle.scss

```diff
@@ -0,0 +1,24 @@
+.typography__label--helper-text {
+    min-height: 0;
+    margin: 0;
+}
+
+.content-toggle {
+  margin-bottom: 4.5rem;
+
+  label {
+    margin-bottom: 0.375rem;
+  }
+}
+
+.content-toggle__label {
+    display: block;
+}
+
+.content-toggle__toggle-btn {
+  cursor: pointer;
+}
+
+.select-field__input {
```

Comment:
I still can't see this class being used by this component. It should either be moved to another file that does or will use it or be deleted from here.

Reply:
This was being amended from the default height but I have now moved it all to the select-field css so it is all in one place. If you look in the console it did show that there was 0.75rem in the select-field and 0.75rem in the select-field__input to make the 1.5rem (24px) that Chris had in his design. This is now all in .select-field and this may need to be changed when we see it in the form.

![Screenshot 2021-05-27 at 08 17 15](https://user-images.githubusercontent.com/745794/119782342-fbf57600-bec3-11eb-97bc-e8fb37bda641.png)

Reply:
I can see that, but what I was trying to convey is that this specific style class is not a part of this component but rather is to do with the usage of the component. Recall that the goal is to create a generic toggle component, which can have any type or number of sub components. It just so happens that our first usage of the component will be to wrap a SelectField component. Hence these style changes should come later.
Also, by adding that class are you not changing the behaviour of all the existing SelectField components in Reviewer? That could have unintended side effects that we don't want to introduce with this commit.